msal/src/main/java/com/microsoft/identity/client/INativeAuthPublicClientApplication.kt

```diff
@@ -151,4 +152,23 @@ interface INativeAuthPublicClientApplication : IPublicClientApplication {
      * @throws MsalClientException if an account is already signed in.
      */
     fun signUpUsingPassword(username: String, password: CharArray, attributes: UserAttributes? = null, callback: NativeAuthPublicClientApplication.SignUpUsingPasswordCallback)
+
+    /**
+     * Reset password for the account starting from a username; Kotlin coroutines variant.
```

Comment:
What do we mean by "starting from"?

Reply:
We start the reset password flow with username. Some flows like signup or signin can be started either with username only or with username and password.